### ISSUE TITLE
Documentation not linked to menu

### ISSUE THREAD

**Issue by whikloj** [Islandora/documentation#1352]
Part of #1040 

When serving mkdocs locally
```
> mkdocs serve
```
It lists files in the `/docs` directory that are not linked from the menu. These are
```
INFO    -  The following pages exist in the docs directory, but are not included in the "nav" configuration:
  - alpaca/alpaca-technical-stack.md
  - contributing/hacking-on-islandora.md
  - contributing/readme_template.md
  - islandora/drupal-bundle-configurations.md
  - migration/fedora-namespace-predicates.md
  - migration/islandora-namespace-predicates.md
  - migration/migration.md
  - mvp/mvp_doc.md
  - technical-documentation/checking-coding-standards.md
  - technical-documentation/drupal-project.md
  - technical-documentation/resizing_vm.md
  - technical-documentation/structure.md
  - user-documentation/access-control.md
  - user-documentation/derivatives.md
  - user-documentation/different_from_7x.md
  - user-documentation/fedora-rdf.md
  - user-documentation/getting-started-with-islandora-claw.md
  - user-documentation/glossary.md
  - user-documentation/intro-to-claw.md
  - user-documentation/intro-to-ld-for-islandora-8.md 
```
They need to be linked or could be deleted.

Additionally it warned about a missing file being referenced.
```
WARNING -  Documentation file 'user-documentation/context.md' contains a link to 'user-documentation/datastreams.md' which is not found in the documentation files. 
WARNING -  Documentation file 'user-documentation/user-intro.md' contains a link to 'user-documentation/datastreams.md' which is not found in the documentation files. 
```

**Comment by seth-shaw-unlv** [comment]
Update: the current docs have a slightly different list:
```
INFO    -  The following pages exist in the docs directory, but are not included in the "nav" configuration:
  - alpaca/alpaca-technical-stack.md
  - contributing/hacking-on-islandora.md
  - contributing/readme_template.md
  - islandora/drupal-bundle-configurations.md
  - migration/fedora-namespace-predicates.md
  - migration/islandora-namespace-predicates.md
  - migration/migration.md
  - mvp/mvp_doc.md
  - technical-documentation/checking-coding-standards.md
  - technical-documentation/drupal-project.md
  - technical-documentation/resizing_vm.md
  - technical-documentation/structure.md
  - user-documentation/access-control.md
  - user-documentation/derivatives.md
  - user-documentation/different_from_7x.md
  - user-documentation/fedora-rdf.md
  - user-documentation/getting-started-with-islandora-claw.md
  - user-documentation/glossary.md
  - user-documentation/intro-to-8.md
  - user-documentation/intro-to-ld-for-islandora-8.md
  - user-documentation/versioning.md 
WARNING -  Documentation file 'user-documentation/collections.md' contains a link to 'assets/collections_admin_content_button.png' which is not found in the documentation files.
```

**Comment by manez** [comment]
I'll work on this today

**Comment by manez** [comment]
One more merge and this one is addressed 😄

**Comment by manez** [comment]
resolved by #1419 and #1421
